roles/ceph-mon/tasks/ceph_keys.yml

```diff
@@ -99,13 +99,35 @@
     - "{{ ceph_keys.get('stdout_lines') | default([]) }}"
   when: cephx
 
+- name: check if libvirt keyring exists
+  command: find /etc/ceph -name "{{ cluster }}.client.libvirt.keyring"
+  changed_when: false
+  register: is_libvirt_keyring_exists
+  always_run: true
+  run_once: true
+  when: cephx
+
+- name: set ceph libvirt keyring permission
+  file:
+    path: "/etc/ceph/{{ cluster }}.client.libvirt.keyring"
+    owner: "ceph"
+    group: "ceph"
+    mode: "0640"
+    command: chown {{ owner }}:{{ group }} {{ path }} ; chmod {{ mode }} {{ path }}
```

Comment:
There is something wrong with your patch here, you included the 'command' module in the file module.
Also, where are those variables declared? owner? group? path? mode?